docs/getting-started/getting-started.md

```diff
@@ -47,6 +47,36 @@ default and completely open source. You can follow along development on
   </em>
 </details>
 
+Once you've downloaded Upstream, start it like this:
+
+#### On Linux:
+
+1. make the package executable with: `chmod +x radicle-upstream-X.X.X.AppImage`
+2. start the app: `./radicle-upstream-X.X.X.AppImage`
+
+#### On macOS:
+
+Open the `radicle-upstream-x.x.x.dmg` package and install Upstream by dragging
+the `radicle-upstream` binary to the `/Applications` folder.
```

Comment:
Is the file called `radicle-upstream`? Should maybe be renamed to `Radicle Upstream` or just `Radicle`

Reply:
I like Radicle Upstream

Reply:
That's correct, the file is called `radicle-upstream` at the moment:

<img width="50%" alt="Screenshot 2020-11-19 at 18 55 47" src="https://user-images.githubusercontent.com/158411/99704561-d6ad2880-2a98-11eb-89c9-efeaf757cbe7.png">

I remember we had an awfully long discussion sometime earlier about this and settled on `radicle-upstream`, did something recently change in our thinking?

Reply:
We have to also keep in mind that the first couple of letters from the app name will be the shortcut in spotlight:
|radicle-upstream|Radicle Upstream|Radicle|Upstream|
|-----------------|------------------|-------|----------|
| <img width="100%" alt="Screenshot 2020-11-20 at 08 53 08" src="https://user-images.githubusercontent.com/158411/99774722-7bb61880-2b0e-11eb-8285-356806a37f4f.png"> | <img width="100%" alt="Screenshot 2020-11-20 at 08 52 43" src="https://user-images.githubusercontent.com/158411/99774710-78bb2800-2b0e-11eb-8716-f439e483eb41.png"> | <img width="100%" alt="Screenshot 2020-11-20 at 08 55 51" src="https://user-images.githubusercontent.com/158411/99774727-7ce74580-2b0e-11eb-8dcb-ff1abc38d6dd.png"> | <img width="100%" alt="Screenshot 2020-11-20 at 08 59 28" src="https://user-images.githubusercontent.com/158411/99774905-b4ee8880-2b0e-11eb-8c8c-905caa693419.png">|

<details>
<summary>package.json</summary>

```diff
diff --git a/package.json b/package.json
index 0958a984..29081f13 100644
--- a/package.json
+++ b/package.json
@@ -1,5 +1,6 @@
 {
   "name": "radicle-upstream",
+  "productName": "Radicle Upstream",
   "version": "0.0.17",
   "author": {
     "name": "radicle team",
@@ -14,10 +15,8 @@
   },
   "build": {
     "appId": "radicle-upstream.monadic.xyz",
-    "files": [
-      "public/**/*",
-      "native/**/*"
-    ],
+    "artifactName": "${name}-${version}.${ext}",
+    "files": ["public/**/*", "native/**/*"],
     "directories": {
       "buildResources": "public"
     },
@@ -32,10 +31,7 @@
       }
     ],
```
</details>

<img width="652" alt="Screenshot 2020-11-20 at 08 54 32" src="https://user-images.githubusercontent.com/158411/99774992-dbacbf00-2b0e-11eb-8e41-f37aeba2873f.png">

Reply:
Updated Upstream: https://github.com/radicle-dev/radicle-upstream/pull/1303.

Reply:
https://github.com/radicle-dev/radicle-docs/pull/26/commits/ddbe85673c84c8ae48ebf4e3c0a2fc9282ac7f79